src/components/ToggleSwitch/index.tsx

```diff
@@ -33,50 +33,39 @@ const styles = StyleSheet.create({
     alignItems: 'center'
   },
   textOn: {
-    fontFamily: 'Inter',
-    fontWeight: '400',
+    fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(12),
     lineHeight: dimen.normalizeDimen(18),
     textAlign: 'center',
     textAlignVertical: 'center',
-    color: COLORS.whiteSmoke,
+    color: COLORS.almostBlackSmoke,
     position: 'absolute',
     top: dimen.normalizeDimen(1),
     left: dimen.normalizeDimen(2),
     width: dimen.normalizeDimen(22),
     height: dimen.normalizeDimen(18)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text when the toggle switch is on (`textOn`) has been changed from `COLORS.whiteSmoke` to `COLORS.almostBlackSmoke`. Ensure that this color change provides sufficient contrast for readability and accessibility. 

```diff
-    color: COLORS.whiteSmoke,
+    color: COLORS.almostBlackSmoke,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->